src/psyclone/tests/psyir/frontend/fparser2_intrinsic_handler_test.py

```diff
@@ -235,7 +235,7 @@ def test_intrinsic_handler_intrinsiccall_onearg(
     routine_symbol = intrinsic_call.routine
     assert isinstance(routine_symbol, RoutineSymbol)
     assert intrinsic_call.routine.name == intrinsic_name
-    assert isinstance(routine_symbol.interface, LocalInterface)
+    assert isinstance(routine_symbol.interface, AutomaticInterface)
```

Comment:
I was going to quibble about this but then I realised it's really the subject of #2102 so it can stay as it is for now.